packages/apps/human-app/frontend/src/modules/worker/oracle-registration/hooks/use-oracle-instructions.tsx

```diff
@@ -0,0 +1,17 @@
+import { useState, useCallback } from 'react';
+import { useGetOracles } from '../../services/oracles';
+
+export function useOracleInstructions(oracleAddress?: string) {
```

Comment:
It can accept just `registrationInstructionsUrl` as a param, which is passed from where `useOracleInstructions` is used - in `useOracleRegistrationFlow`